src/audio.c

```diff
@@ -1119,7 +1128,7 @@ static int aurx_stream_decode(struct aurx *rx, bool marker,
 
 
 /* Handle incoming stream data from the network */
-static void stream_recv_handler(const struct rtp_header *hdr,
+static int stream_recv_handler(const struct rtp_header *hdr,
 				struct rtpext *extv, size_t extc,
 				struct mbuf *mb, unsigned lostc, void *arg)
```

Comment:
can we keep this to have return type void ?

I prefer void in callback handlers, because the handler code cannot fail
hence it is easier to write the code.